src/content/posts/whats-possible-with-melange.mdx

```diff
@@ -0,0 +1,202 @@
+---
+title: "What's possible with Melange"
+publishedAt: "2023-12-18"
+tags:
+  - "Melange"
+  - "dune"
+  - "OCaml"
+  - "Reason"
+---
+
+## Introduction
+
+[Melange](https://melange.re) is a backend for the OCaml compiler that emits readable, modular and highly performant JavaScript code. It’s integrated with all OCaml-related tooling such as opam to install packages and dune as a build system.
+
+This blog post is an overview of the capabilities of Melange and dune together, what's possible to do now and it wasn't with the previous solution (ReScript v9).
+
+This blog post isn't a direct comparison between Melange v2 and ReScript v9 (also know as BuckleScript) compilers, it wouldn’t be fair since ReScript v9 is not the latest version and their goals were not designed to solve any of the problems Melange solves.
+
+Melange started as a fork of BuckleScript but focusing in OCaml compatibility while ReScript continued the integration with JavaScript ecosystem.
+
+## Works with the latest versions of OCaml
+
+One blocker for some OCaml developers to use ReScript is the version of the OCaml compiler being stuck in 4.06, a few versions behind latest (5.1) OCaml.
```

Comment:
it's probably more clear to mention "typechecker" instead of compiler, since ReScript has since vendored the 4.06 typechecker but can be built with newer OCaml versions.